### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: src/exporter.js [hunk 1]
```diff
@@ -21,13 +20,10 @@ function Exporter (hash, dagService, options) {
     return new Exporter(hash, dagService, options)
   }
 
-  // Sanitize hash.
+  // Sanitize hash
   if (!isIPFS.multihash(hash)) {
     throw new Error('not valid multihash')
   }
-  if (Buffer.isBuffer(hash)) {
-    hash = bs58.encode(hash)
```
Comment (noffle) [inline src/exporter.js]: Why was this removed? It breaks functionality. :(  Fixed in https://github.com/ipfs/js-ipfs-unixfs-engine/pull/52

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# IPFS unixFS Engine
+IPFS unixFS Engine
```
Comment (RichardLitt) [inline README.md]: What's wrong with #? It's the same, in rendering.

File: src/exporter.js (hunk 1 above)
Comment (diasdavid) [inline src/exporter.js]: Thank you for catching it, tests didn't detect it

File: README.md (hunk 2 above)
Comment (diasdavid) [inline README.md]: I've been (since forever) doing `====` for the title, and now it is just autopilot.

Comment (RichardLitt) [inline README.md]: Ok. Let's not care.
